ext_emconf.php

```diff
@@ -1,5 +1,6 @@
 <?php
-$EM_CONF[$_EXTKEY] = [
+
+$EM_CONF['cache_automation'] = [
```

Comment:
that is not ok for the TER we need to use the $_EXTKEY variable
```suggestion
$EM_CONF[$_EXTKEY] = [
```

Reply:
We need to Change the Version constraint in Line 20
```
    'typo3' => '8.7.0-10.4.99',
```